library/traefik

```diff
@@ -1,40 +1,46 @@
-Maintainers: Emile Vauge <emile@vauge.com> (@emilevauge), Vincent Demeester <vincent@sbr.pm> (@vdemeester), Ludovic Fernandez <ludovic@containo.us> (@ldez), Julien Salleyron <julien@containo.us> (@juliens), Nicolas Mengin <nicolas@containo.us> (@nmengin), Michael Matur <michael@containo.us> (@mmatur)
-GitRepo: https://github.com/containous/traefik-library-image.git
+Maintainers: Emile Vauge <emile@vauge.com> (@emilevauge), Vincent Demeester <vincent@sbr.pm> (@vdemeester), Ludovic Fernandez <ludovic@traefik.io> (@ldez), Julien Salleyron <julien@traefik.io> (@juliens), Romain Tribotté <romain.tribotte@traefik.io> (@rtribotte), Michael Matur <michael@traefik.io> (@mmatur)
 
-Tags: v2.3.0-rc5-windowsservercore-1809, 2.3.0-rc5-windowsservercore-1809, v2.3-windowsservercore-1809, 2.3-windowsservercore-1809, picodon-windowsservercore-1809
+Tags: v2.3.0-rc6-windowsservercore-1809, 2.3.0-rc6-windowsservercore-1809, v2.3-windowsservercore-1809, 2.3-windowsservercore-1809, picodon-windowsservercore-1809
+GitRepo: https://github.com/traefik/traefik-library-image.git
 Architectures: windows-amd64
-GitCommit: e72df1d91ffa11fd0dc93330dbb61da90f7758a0
+GitCommit: 9d823fd7bcc724b09bd9945803f28fc4ae98627f
 Directory: windows/1809
 Constraints: windowsservercore-1809
 
-Tags: v2.3.0-rc5, 2.3.0-rc5, v2.3, 2.3, picodon
+Tags: v2.3.0-rc6, 2.3.0-rc6, v2.3, 2.3, picodon
+GitRepo: https://github.com/traefik/traefik-library-image.git
 Architectures: amd64, arm32v6, arm64v8
-GitCommit: e72df1d91ffa11fd0dc93330dbb61da90f7758a0
+GitCommit: 9d823fd7bcc724b09bd9945803f28fc4ae98627f
 Directory: alpine
 
 Tags: v2.2.11-windowsservercore-1809, 2.2.11-windowsservercore-1809, v2.2-windowsservercore-1809, 2.2-windowsservercore-1809, chevrotin-windowsservercore-1809, windowsservercore-1809
+GitRepo: https://github.com/containous/traefik-library-image.git
```

Comment:
I'm slightly confused as to why https://github.com/traefik/traefik-library-image isn't either a github "move" (and so gets redirects automatically and issues/PRs move) or doesn't at least contain all commits from https://github.com/containous/traefik-library-image (basically a fork but not a "github fork" so that it doesn't have a parent relationship to `containous/traefik-library-image`).

Reply:
https://github.com/traefik/traefik-library-image is a really huge repository because in the past we stored binaries in this repository.

We don't want to rewrite history or keep the huge amount of data of the previous repo, so we created a new repo.

Reply:
In the future, we will drop the previous repo from this manifest, but one step at a time.

Reply:
Sounds good. Thank you, that was helpful.